src/CMakeLists.txt

```diff
@@ -154,7 +150,9 @@ endif ( RADIUM_IO_IS_INTERFACE )
 
 
 add_dependencies( ${io_target} ${core_target} ${engine_target} )
-target_include_directories(${io_target} PUBLIC ${RADIUM_SRC_DIR})
+target_include_directories(${io_target}
+    PUBLIC ${RADIUM_SRC_DIR}
```

Comment:
no need for Eigen?

Reply:
We do not necessarily need it. Actually this is add by each IO support (eg ``AssimpLoader/CMakeLists.txt``
I think that each IO-support must define the minimal set of include directory, that is not the case actually.  See commit about that.